utils/migration_tools/backup_neptune/README.md

```diff
@@ -0,0 +1,94 @@
+# Back up run metadata from Neptune
+
+You can use this script to download run metadata from Neptune to your system. Note that this script is currently in beta, and we welcome your feedback and contributions to help improve it.
+
+## Prerequisites
+
+Before using this script, make sure you have the Neptune environment variables set up. For instructions, see the [documentation](https://docs.neptune.ai/setup/setting_credentials/).
+
+## Instructions
+
+To use the script, follow these steps:
+
+1. The `bulk_download_metadata.py` script will generate run logs in the same folder as itself. You can modify this location by editing the `logging.basicConfig()` function.
```

Comment:
My point was that this is not exactly a step, but information and some optional preconfig. I'd just move this to the top of the section and have the ordered steps start from running the script.

Reply:
Lemme know if it looks better now